src/main/java/com/google/api/codegen/transformer/java/JavaGapicSurfaceTestTransformer.java

```diff
@@ -78,6 +85,27 @@ public JavaGapicSurfaceTestTransformer(GapicCodePathMapper javaPathMapper) {
     return views;
   }
 
+  public Iterable<Interface> getGrpcInterfacesToMock(Model model, ApiConfig apiConfig) {
```

Comment:
It should be private. Silly eclipse, automatically assuming public. I'll fix it.